routes/api/contacts.js

```diff
@@ -1,25 +1,160 @@
-const express = require('express')
+import express from 'express';
+import Joi from 'joi';
 
-const router = express.Router()
+import {
+  listContacts,
+  getContactById,
+  removeContact,
+  addContact,
+  updateContact,
+} from './../../models/contacts.js';
+
+export const router = express.Router();
+const sendErrorResponse = (res, status, message) => {
```

Comment:
In my opinion this method is not needed. It makes code a little bit less readable for me but if its ok for you, no problem :).